app/assets/stylesheets/components/_location_details_table.scss

```diff
@@ -0,0 +1,3 @@
+.location-details_table tr td:first-child {
```

Comment:
consistent underscore 
and use
.location-details-table__fixed-width-column - on td field only